src/reflection/FStar.Reflection.Basic.fs

```diff
@@ -208,7 +208,7 @@ let rec inspect_ln (t:term) : term_view =
         | BU.Inl bv -> Tv_Let (true, lb.lbattrs, bv, lb.lbdef, t2)
         end
 
-    | Tm_match (t, brs) ->
+    | Tm_match (t, _, brs) ->  //AR: TODO: expose the annotation in the reflection API
```

Comment:
Would be great to address this now, otherwise we'll forget and not do it until a user complains about it